subjects/scripting/pyFuncNames.reko/sum.c

```diff
@@ -7,7 +7,7 @@
 // 10001000: Register (ptr32 Eq_n) sum_wrapper(Stack (ptr32 Eq_n) ptrArg04, Stack (ptr32 Eq_n) ptrArg08)
 PyObject * sum_wrapper(PyObject * ptrArg04, PyObject * ptrArg08)
 {
-	PyObject * eax_n = PyArg_ParseTuple(ptrArg08, "ii:sum", fp - 0x04, fp - 0x08);
```

Comment:
As discussed earlier, the declarations of the two new variables can be done in a future commit.